api/v1alpha1/releaselink_webhook.go

```diff
@@ -29,12 +30,19 @@ func (r *ReleaseLink) SetupWebhookWithManager(mgr ctrl.Manager) error {
 		Complete()
 }
 
-// +kubebuilder:webhook:path=/mutate-appstudio-redhat-com-v1alpha1-releaselink,mutating=true,failurePolicy=fail,sideEffects=None,groups=appstudio.redhat.com,resources=releaselinks,verbs=create;update,versions=v1alpha1,name=mreleaselink.kb.io,admissionReviewVersions=v1
+// +kubebuilder:webhook:path=/mutate-appstudio-redhat-com-v1alpha1-releaselink,mutating=true,failurePolicy=fail,sideEffects=None,groups=appstudio.redhat.com,resources=releaselinks,verbs=create,versions=v1alpha1,name=mreleaselink.kb.io,admissionReviewVersions=v1
```

Comment:
for my understanding....why was "update" removed?

Reply:
`update` being there is the default, but we had this webhook turned off before (as you can see by the function being empty). To my best knowledge, our desired behavior is to add a label if it doesn't exist when a ReleaseLink is created. I don't believe we have any desired behavior on ReleaseLink updates, right?

It would have made more sense if this PR introduced this entire line (containing just `create`) but for whatever reason we already had the code in our operator, but we didn't have it doing anything

Reply:
Thinking about it more in detail...

I guess someone could create a ReleaseLink and our webhook would add the label.
then he/she could edit it and remove the label.
We'd want to catch that too, no?

Reply:
You tell me. My understanding was that integration controller should create a Release if the label exists and == true OR doesn't exist. So the only way to be set to `manual` mode is to create your ReleaseLink with the label set to false, or set it to false at some point after it is created before the integration controller runs. Do we want to add it set it true if the label is removed entirely? This would eliminate the "doesn't exist" scenario from my second sentence here.

@davidmogar thoughts?